concent_api/core/payments/mock.py

```diff
@@ -33,43 +18,20 @@ def get_list_of_transactions(_oldest_payments_ts = None, current_time = None, _t
             payment_ts = current_time - settings.PAYMENT_DUE_TIME
```

Comment:
Same here. @dybi

Reply:
@rwrzesien, I don't think I need it

Reply:
As above I change mock backend to return null values